### PR TITLE
BaSP-TG: Project endpoints added

### PR DESCRIPTION
**Pull request by LucianoFranchino**
Endpoints:
- Create Project
- Get all
- Get by Id

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,12 +3,17 @@ import express from 'express';
 
 // use "require" to import JSON files
 const admins = require('./data/admins.json');
+const callRouter = require('./resources/projects');
 
 const app = express();
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 6000;
```
Comment (ignacioaristo) [inline src/index.js:9]: let keep the port in 3000.

File: src/index.js [hunk 2]
```diff
@@ -3,12 +3,17 @@ import express from 'express';
 
 // use "require" to import JSON files
 const admins = require('./data/admins.json');
+const callRouter = require('./resources/projects');
```
Comment (ignacioaristo) [inline src/index.js:6]: we could be more specific with this name, for e.g: projectsRouter

File: src/index.js [hunk 3]
```diff
@@ -3,12 +3,17 @@ import express from 'express';
...
 
 const app = express();
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 6000;
 
 app.use(express.json());
 
+app.use(express.urlencoded({ extended: false }));
```
Comment (ignacioaristo) [inline src/index.js:13]: Is this line necessary for now?

**Review by ignacioaristo — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (LucianoFranchino) [reply src/index.js:9]: Resolved!

File: src/index.js (hunk 2 above)
Comment (LucianoFranchino) [reply src/index.js:6]: Resolved!

File: src/index.js (hunk 3 above)
Comment (LucianoFranchino) [reply src/index.js:13]: I think that line was already in the file, I deleted it anyway!

**Review by ignacioaristo — APPROVED** [review]
Good job

**Pull request merged**